src/components/layout/NavHeader.module.css

```diff
@@ -0,0 +1,68 @@
+nav {
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  height: 30px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 40px;
+  margin-inline: auto;
+  padding-block: 30px;
+  background-color: rgb(5, 146, 202);
+  border-bottom: 1px solid rgb(180, 172, 172);
+  padding-inline: 5%;
+}
```

Comment:
- [ ] I like the fact that you have added the back button as requested by the previous reviewer, but the requirements for this project say you should follow all the design guidelines like Typography: font face, size, and weight,
Layout: composition and space between elements. Here is the difference in designs below: 

## Yours

![mobile1](https://user-images.githubusercontent.com/53124548/210966565-e0477a01-723a-4cec-9c21-f17a697767ee.png)

## Original

![mobile](https://user-images.githubusercontent.com/53124548/210966626-94a05a74-de1a-42d4-a3da-6df5e62a965b.png)

- [ ] [OPTIONAL] You can do the same with the home page to be consistency.